File: 01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/NewHouse.java
import java.util.Scanner;

public class NewHouse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String flowersType = scanner.nextLine();
        int flowersNumber = Integer.parseInt(scanner.nextLine());
        int budget = Integer.parseInt(scanner.nextLine());
        double flowerPrice = 0;
        switch (flowersType) {
            case "Roses":
                flowerPrice = 5;
                break;

            case  "Dahlias":
                flowerPrice = 3.8;
                break;

            case "Tulips":
                flowerPrice = 2.8;
                break;

            case  "Narcissus":
                flowerPrice = 3;
                break;

            case  "Gladiolus":
                flowerPrice = 2.5;
                break;

            default:
                break;
        }

        double totalPrice = flowersNumber * flowerPrice;
        if (flowersType.equals("Roses") && flowersNumber > 80) {
            totalPrice -= (totalPrice * 0.1);
        } else if (flowersType.equals("Dahlias") && flowersNumber > 90) {
            totalPrice -= (totalPrice * 0.15);
        } else if (flowersType.equals("Tulips") && flowersNumber > 80) {
            totalPrice -= (totalPrice * 0.15);
        } else if (flowersType.equals("Narcissus") && flowersNumber < 120) {
            totalPrice += (totalPrice * 0.15);
        } else if (flowersType.equals("Gladiolus") && flowersNumber < 80) {
            totalPrice += (totalPrice * 0.2);
        }

        if (budget >= totalPrice) {
            System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.", flowersNumber, flowersType, budget-totalPrice);
        } else {
            System.out.printf("Not enough money, you need %.2f leva more.", totalPrice-budget);
        }
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/NewHouse.js
function flowers(params) {
    let type = params[0];
    let count = Number(params[1]);
    let budget = Number(params[2]);

    let price = 0.0;
    if (type === "Roses") {
        price = count * 5.00;
        if (count > 80) {
            price -= (price * 0.1);
        }
    } else if (type == "Dahlias") {
        price = count * 3.80;
        if (count > 90) {
            price -= (price * 0.15);
        }
    } else if (type == "Tulips") {
        price = count * 2.80;
        if (count > 80) {
            price -= (price * 0.15);
        }
    } else if (type === "Narcissus") {
        if (count < 120) {
            price = count * (3.00 + (3.00 * 0.15));
        } else {
            price = count * 3;
        }
    } else if (type === "Gladiolus") {
        if (count < 80) {
            price = count * (2.50 + (2.50 * 0.20));
        } else {
            price = count * 2.50;
        }
    }

    if (budget >= price) {
        console.log(`Hey, you have a great garden with ${count} ${type} and ${(budget-price).toFixed(2)} leva left.`);
    } else {
        console.log(`Not enough money, you need ${(price - budget).toFixed(2)} leva more.`);
    }
}

flowers(["Roses","55","250"]);
flowers(["Tulips","88","260"]);
flowers(["Narcissus","119","360"]);